
This remove conditional displaying of "00x00 ~ subtitle" in video list when 00x00 is present in filename
as this results displaying blank video title in list when filename has 00x00 but don't have subtitle


diff --speed-large-files --no-dereference --minimal -Naur Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/video-ui.xml Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/video-ui.xml
--- Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/video-ui.xml	2022-04-26 01:09:17.000000000 +0200
+++ Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/video-ui.xml	2022-11-09 12:56:33.390527590 +0100
@@ -222,18 +222,9 @@
                     <textarea name="buttontext">
                         <alpha>0</alpha>
                     </textarea>
-                    <!-- use series/episode number to switch displayed text -->
-                    <textarea name="00x00">
-                        <alpha>0</alpha>
-                    </textarea>
-                    <textarea name="title" from="baseitemtext" depends="!00x00">
-                        <area>5,0,-37,100%</area>
-                        <template>%TITLE| %%~ |SUBTITLE%</template>
-                        <alpha>100</alpha>
-                    </textarea>
-                    <textarea name="subtitle" from="baseitemtext" depends="00x00">
+                    <textarea name="title" from="baseitemtext">
                         <area>5,0,-37,100%</area>
-                        <template>%00x00| %%~ |SUBTITLE%</template>
+                        <template>%TITLE| %%~ episode |EPISODE| %%~ |SUBTITLE%</template>
                         <alpha>100</alpha>
                     </textarea>
                     <statetype name="watchedstate" from="basewatchedstate">
